plugins/login/login.js

```diff
@@ -44,9 +91,23 @@ module.exports = (config) => ({
             });
 
             request.cookieAuth.set(profile);
+            const userConfig = {
+                id: hashr.sha1(username),
```

Comment:
@stjohnjohnson  How about in a separate PR since I want to update them all at once (for other plugins as well)